demo-scalability.md

```diff
@@ -1,8 +1,176 @@
 ---
 title: Scalability
-summary: 
+summary: Use a local cluster to explore how CockroachDB automatically rebalances data as you scale.
 toc: false
-feedback: false
 ---
 
-Coming soon: Use a local cluster to explore how easy it is to scale CockroachDB with your data.
+This page walks you through a simple demonstration of how CockroachDB automatically rebalances data as you scale. Starting with a 3-node local cluster, you'll lower the maximum size for a single range, the unit of data that is replicated in CockorachDB. You'll then download and connect the `block_writer` example program, which will continuously insert a large amount of data into your cluster, and watch the replica count quickly increase as ranges split. You'll then add 2 more nodes and watch how CockroachDB automatically rebalances replicas to efficiently use all available capacity.
```

Comment:
s/which will continuously insert a large amount of data/which continuously inserts data/

This sounds more precise:

s/efficiently use all available/spread the load across the available/

"efficiently" is kind of marketing-y